packages/framework/src/Resources/config/packages_registry.yaml

```diff
@@ -2,3 +2,19 @@ parameters:
     shopsys.packages.registry:
         - { name: 'framework', path: '%shopsys.framework.root_dir%', namespace: 'Shopsys\FrameworkBundle', app_namespace: 'App' }
         - { name: 'frontend-api', path: '%shopsys.framework.root_dir%/../frontend-api', namespace: 'Shopsys\FrontendApiBundle', app_namespace: 'App\FrontendApi' }
+        - { name: 'article-feed-luigis-box', path: '%shopsys.framework.root_dir%/../article-feed-luigis-box', namespace: 'Shopsys\ArticleFeed\LuigisBoxBundle', app_namespace: 'App\ArticleFeed\LuigisBox' }
```

Comment:
I am afraid that:
- it will be hard to maintain (I can imagine we will keep forgetting about this when adding a new package)
- it will be hard to edit from a project - there are no docs for this feature so on a project, I will not know about this config and I can easily mismatch the `app_namespace` setting (I can extend a package in a different namespace then is set here)

Reply:
I am sorry, I have no better suggestions at the moment, but we can brainstorm it

Reply:
I have reworked it slightly and added docs for it